### PR TITLE
Dmitri/cache

### PR DESCRIPTION
**Pull request by a-palchikov**
Adds in-memory cache that persists the last cluster status.
Also drops sqlite support.

### CODE REVIEW

File: agent/backend/inmemory/inmemory.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package inmemory
+
+import pb "github.com/gravitational/satellite/agent/proto/agentpb"
+
+// New creates a new instance of cache
+func New() *cache {
```
Comment (klizhentas) [inline agent/backend/inmemory/inmemory.go]: is it multi-threaded?

Comment (klizhentas) [inline agent/backend/inmemory/inmemory.go]: could be called by multiple goroutines?

**Comment by r0mant** [comment]
What about influxdb backend? Do we want to keep it?

**Comment by klizhentas** [comment]
cache does not seem to be protected for concurrent access. can you access it via multiple goroutines @a-palchikov

**Comment by a-palchikov** [comment]
I think we should influxdb - even if it's not used currently, it serves as an example of implementing a backend for satellite.

**Pull request merged**